composer.json

```diff
@@ -24,7 +24,7 @@
         "issues": "https://github.com/oscarotero/Embed/issues"
     },
     "require": {
-        "php": "^5.5|^7.0",
+        "php": "^5.6|^7.0",
```

Comment:
This is a breaking change and requires a major version. Not sure about this...

Reply:
The latest `php-5.5` version seems to be failed on Travis CI build.

It cause this problem that mentions on this [issue](https://github.com/travis-ci/travis-ci/issues/6339).

You can take a look at this [Travis CI build log](https://travis-ci.org/peter279k/Embed/builds/495333147) to know more details about that.